src/tests/ftest/SConscript

```diff
@@ -10,7 +10,7 @@ def scons():
 
     env.Install(ftest_install_dir, Glob('*.*'))
 
-    dirs = ['aggregation', 'fault_injection', 'checksum',
+    dirs = ['aggregation', 'fault_injection', 'cat_recovery', 'checksum',
```

Comment:
How do you feel about simplifying the new directory name to `recovery`?

Reply:
Done